gossip/simulation/network.go

```diff
@@ -76,13 +76,12 @@ func NewNetwork(nodeCount int, networkType string,
 	} else {
 		bootstrap = addrs[:3]
 	}
-
 	nodes := make([]*Node, nodeCount)
 	for i := 0; i < nodeCount; i++ {
-		node := gossip.New(rpcContext, gossipInterval, gossipBootstrap)
+		// Pass an empty string to gossipBootstrap and set it later.
+		node := gossip.New(rpcContext, gossipInterval, "")
```

Comment:
The comment is a bit confusing because `gossip.gossipBootrap` is not the same as `gossip.bootstraps`. The naming of those fields is confusing as well. Perhaps @spencerkimball can clarify.